### PR TITLE
New Schizo Snippets

### PR DESCRIPTION
**Pull request by Empyrean-Heaven**
#### Summary

SUMMARY: Content "Some new snippets for schizophrenia"

#### Purpose of change

"I'm not crazy, you are!" This PR adds a few more schizo snippets to spice things up.

#### Describe the solution

Either A: add some pretty crazy sounding things like now, or B: become schizophrenic myself and experience the TRUE Cataclysm and attempt to port it into the game.

#### Describe alternatives you've considered

Ooga Booga. Small brain, no insanity.

#### Testing

Some simple JSON changes.

#### Additional context

So what are we?

### CODE REVIEW

File: data/json/snippets/schizophrenia.json [hunk 1]
```diff
@@ -23,7 +23,16 @@
       "This place is dangerous, you shouldn't be here.",
       "What are you doing out here?",
       "That's not true, is it?",
-      "Are you hurt?"
+      "Are you hurt?",
+      "I awaken when your slumber.",
```
Comment (chaosvolt) [inline data/json/snippets/schizophrenia.json:27]: I assume this was meant to be `I awaken when you slumber.` instead?

**Comment by chaosvolt** [comment]
Probably will want double spaces after sentences on these too.

**Comment by Empyrean-Heaven** [comment]
Double spaces?

**Comment by chaosvolt** [comment]
Description and other text in Cata tends to always have two spaces between sentences instead of one.

File: data/json/snippets/schizophrenia.json [hunk 2]
```diff
@@ -23,7 +23,16 @@
       "This place is dangerous, you shouldn't be here.",
       "What are you doing out here?",
       "That's not true, is it?",
-      "Are you hurt?"
+      "Are you hurt?",
+      "I awaken when you slumber.",
+      "Was that noises? I think I heard noises.",
+      "I saw something up ahead. I promise this time.",
```
Comment (chaosvolt) [inline data/json/snippets/schizophrenia.json:29]:
```suggestion
      "Was that noises?  I think I heard noises.",
      "I saw something up ahead.  I promise this time.",
```

File: data/json/snippets/schizophrenia.json [hunk 3]
```diff
@@ -23,7 +23,16 @@
...
+      "Who cried wolf!?",
+      "The trees are talking behind your back.",
+      "And thus we live to struggle another day.",
+      "Any man, no matter how hard, can bear his burdens for one day.",
+      "Did you sleep well last night? And the night before that? I was with you from the very beginning.",
```
Comment (chaosvolt) [inline data/json/snippets/schizophrenia.json:34]:
```suggestion
      "Did you sleep well last night? And the night before that?  I was with you from the very beginning.",
```

File: data/json/snippets/schizophrenia.json [hunk 4]
```diff
@@ -39,7 +48,9 @@
       "Wait, that %1$s needs to die!",
       "Go kill that %1$s!",
       "Look at that %1$s!",
-      "That %1$s doesn't deserve to live!"
+      "That %1$s doesn't deserve to live!",
+      "That %1$s looked at you funny. Kill it. Now.",
+      "That %1$s seems to be smiling at you. Go say hi for me."
```
Comment (chaosvolt) [inline data/json/snippets/schizophrenia.json:53]:
```suggestion
      "That %1$s looked at you funny.  Kill it.  Now.",
      "That %1$s seems to be smiling at you.  Go say hi for me."
```

File: data/json/snippets/schizophrenia.json [hunk 5]
```diff
@@ -53,7 +64,9 @@
       "You look hurt, did I do that?",
       "Are you supposed to be bleeding?",
       "You're not going to die, are you?",
-      "Kill a few more before you bleed out!"
+      "Kill a few more before you bleed out!",
+      "Let me have a taste of your blood.",
+      "Thrust me deeper into the wound. You know that this is the only way out right?"
```
Comment (chaosvolt) [inline data/json/snippets/schizophrenia.json:69]:
```suggestion
      "Thrust me deeper into the wound.  You know that this is the only way out right?"
```

File: data/json/snippets/schizophrenia.json [hunk 6]
```diff
@@ -96,7 +114,10 @@
       "You gain the sudden realization that you must have been responsible for the Cataclysm.",
       "You get the odd feeling that your thoughts are not your own.",
       "You feel as if everyone in the world must be playing a sick prank on you.",
-      "You are being watched… by THEM."
+      "You are being watched… by THEM.",
+      "You sense a vast aura envelop the landscape, and the skies seem to turn a bleeding edge. Then, a thunderclap is heard in the distance, and eveything becomes normal once more. For now.",
+      "You feel your innards churn as you suddenly develop the urge to tear grass out of its roots.",
+      "You begin to sweat uncontrollably as your hands start to shake. It feels as if something is shaking them violently."
```
Comment (chaosvolt) [inline data/json/snippets/schizophrenia.json:120]:
```suggestion
      "You sense a vast aura envelop the landscape, and the skies seem to turn a bleeding edge.  Then, a thunderclap is heard in the distance, and everything becomes normal once more.  For now.",
      "You feel your innards churn as you suddenly develop the urge to tear grass out of its roots.",
      "You begin to sweat uncontrollably as your hands start to shake.  It feels as if something is shaking them violently."
```

File: data/json/snippets/schizophrenia.json [hunk 7]
```diff
@@ -108,7 +129,8 @@
       "You gain the sudden realization that you must have been responsible for the Cataclysm.",
       "You feel that this must be a global reality show, in which you are the star.",
       "You now know that you are a secret agent, kept alive to continue serving the government.",
-      "You feel in full control of the situation."
+      "You feel in full control of the situation.",
+      "You attained perfection a long time ago. This world is but a mirage of your own making. Thus, this is a perfect world."
```
Comment (chaosvolt) [inline data/json/snippets/schizophrenia.json:133]:
```suggestion
      "You attained perfection a long time ago.  This world is but a mirage of your own making.  Thus, this is a perfect world."
```

File: data/json/snippets/schizophrenia.json [hunk 8]
```diff
@@ -142,7 +165,8 @@
       "\"I had to do it!\"",
       "\"They made me do it!\"",
       "\"What are you!?\"",
-      "\"I should never have trusted you!\""
+      "\"I should never have trusted you!\"",
+      "\"RUN. NOW. IT MAY NOT BE TOO LATE.\""
```
Comment (chaosvolt) [inline data/json/snippets/schizophrenia.json:169]:
```suggestion
      "\"RUN.  NOW.  IT MAY NOT BE TOO LATE.\""
```

**Comment by Empyrean-Heaven** [comment]
Wow. Thanks for all of this. I never knew double spaces were the norm in Cata. No wonder everything seems evenly spaced out. Also, thank your for correcting some of my spelling. I really appreciate it.

**Comment by chaosvolt** [comment]
No worries. It's always struck me as weird, basically the only situation I've seen online where double spaces are the norm.

**Review by chaosvolt — APPROVED** [review]

**Pull request merged**